sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_check_v1/query.sql

```diff
@@ -0,0 +1,48 @@
+WITH dates AS (
+  -- Generate the date range to check for missing data
+  SELECT
+    *
+  FROM
+    UNNEST(GENERATE_DATE_ARRAY('2021-05-01', '@submission_date', INTERVAL 1 DAY)) AS date
+),
+partition_info AS (
+  SELECT
+    date
+  FROM
+    dates
+  LEFT JOIN
+    (
+    -- Based on the information we have about existing partitions
+    -- determine which partitions are either missing or have 0 rows.
+    -- This is cheaper than querying the table every day to get the row count.
+      SELECT
+        *
+      FROM
+        `contextual_services_derived.INFORMATION_SCHEMA.PARTITIONS`
+      WHERE
+        table_name = 'event_aggregates_v1'
+        AND total_rows > 0
```

Comment:
So this checks for partitions that are entirely missing or have 0 rows of data (we could also do something like `total_rows > 10`). Not sure if that is the check we want, it would have at least caught the backfill issues we saw

Reply:
Is it common that a backfill issue can cause <10 rows to show up instead of 100% missingness?

Reply:
No, that would not indicate a backfill issue, but some issue with the downstream data